docs/api/features/networks.md

```diff
@@ -26,3 +31,10 @@ is not supported at this time.
 
 If fabric networking is not enabled, all docker containers are provisioned with
 a nic on the 'external' network by default.
+
+
+## Related
+
+- [`sdc-fabric vlan`](https://apidocs.joyent.com/cloudapi/#CreateFabricVlan) and `POST /my/fabrics/default/vlans` in CloudAPI
+- [`sdc-fabric network`](https://apidocs.joyent.com/cloudapi/#CreateFabricNetwork) and `POST /my/fabrics/default/vlans/:id/networks` in CloudAPI
+- [`sdc-fabric networki set-default`](https://apidocs.joyent.com/cloudapi/#SetDefaultNetwork) and `PUT /my/config` in CloudAPI
```

Comment:
None of these links are valid at this time. Also, presumably we'll want to link to the user guide. @misterbisson do you know what the URL for that will be?